gui.lua

```diff
@@ -1791,6 +1791,29 @@ function GUI:Init()
                     StaticPopup_Show("RAIDLEDGER_DELETE_ITEM")
                 else
                     ChatEdit_InsertLink(entry["detail"]["item"])
+
+                    if IsShiftKeyDown() then
+                        local ledger = Database:GetCurrentLedger()
+                        if idx == #ledger["items"] then
+                            return
+                        end
+                        local nextEntry = ledger["items"][idx+1]
+                        ledger["items"][idx] = nextEntry
+                        ledger["items"][idx+1] = entry
+                        GUI:UpdateLootTableFromDatabase()
+                    end
+
+                    if IsAltKeyDown() then

```

Comment:
what if i press shift alt simultaneously?

Reply:
didn't consider about that...